distributions/demo-google-deployment/api/build.gradle

```diff
@@ -93,9 +113,8 @@ task createApiServerDockerfile(type: Dockerfile) {
     from "gcr.io/google-appengine/openjdk:8"
     exposePort 8080 // Port the API server is accessed from
     copyFile("build/libs/api-all.jar", "/app/api.jar")
-    // TODO: make debug flag driven
     defaultCommand("java",
-            "-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005",
+            "-Xrunjdwp:transport=dt_socket,server=y,suspend=n",
```

Comment:
Should these be driven by a flag?

Reply:
I have a TODO to do so for the dockerizeLocal task. I left it out of this task since it's for production and we shouldn't expose a debug port.